### DISCUSSION TITLE
How to use unpoly to track changes in a dropdown menu?

### DISCUSSION THREAD

**Discussion by nmcdtu** [unpoly/unpoly#635] [Q&A] [answered]
I wish to use `unpoly` to track changes in a dropdown menu. A segment of my HTML code appears as follows:
```html
<div id="sidebar" style="overflow-y: auto;">
    <div id="prep_page">
        <div class="text-4xl font-bold mt-6">No. bands:
          <select id="no_bkps_drop">
            
                <option value="1">1</option>
            
                <option value="2">2</option>
            
                <option value="3">3</option>
            
                <option value="4">4</option>
            
                <option value="5">5</option>
            
          </select>
        </div>
    </div>
</div>
```

I want to track changes in the dropdown menu above `no_bkps_drop`. My current method failed without producing any errors; that is, the click on the dropdown `no_bkps_drop` was not detected.
```js
var form = document.getElementById('sidebar');
up.on(form, '#no_bkps_drop', 'change', function(event) {
    // My code
}
```

The following workaround I implemented is functional, but it's not precisely what I need. Currently, it activates every time I click the `no_bkps_drop` dropdown, whereas I want it to execute only when the dropdown's value changes:
```js
up.on(form, 'click', function(event) {
  if (event.target.matches('#no_bkps_drop')) {
    // My code
  }
}
```

I would be very grateful if someone could suggest a solution to my problem.
Thank you very much in advance!

**Comment by triskweline** [comment, marked as answer]
You got the argument order mixed up. It needs to be:

```js
up.on(form, 'change', '#no_bkps_drop', function(event) {
    // My code
}
```

**Reply by nmcdtu** [reply]
Thank you!
